packages/generator/src/commerceTemplateHelper.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { model } from "@commerce-apps/raml-toolkit";
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const templateHelpers = require("../src/templateHelpers");
+
+/**
+ * Given a type, prefix it with the namespace "types."
+ * @param type - in string format to be prefixed
+ * @returns the type prefixed by types.
+ */
+export function addNamespacePrefixToType(type: string): string {
+  if (!type) {
+    return type;
+  }
+  const prefix = "types.";
+  const types = type.split("|");
```

Comment:
I am confused about the flow here? Is this happening after the rendering of the list of types?  Wouldn't this be simpler if it happened prior to that union type listing?